util/session/sessionmanager.go

```diff
@@ -43,6 +43,12 @@ const (
 	badUserError       = "Bad local superuser username"
 )
 
+// JwtToken the metadata of a token
+type JwtToken struct {
```

Comment:
This datastructure is redundant. The string returned by SessionManager.Create() is already a JWT token with information about when it was issued.

Reply:
Fixed